testsuite/integration/clustering/pom.xml

```diff
@@ -2306,7 +2309,7 @@
                                 </configuration>
                             </execution>
                             <execution>
-                                <id>ts.surefire.clustering.ha-infinispan-server</id>
+                                <id>ts.surefire.clustering.ha-no-reuse-forks</id>
```

Comment:
@rachmatowicz This test doesn't pass for me with -Dts.ee9 so it needs to be added to the exclude list a bit below this.

I think the new test has the EJB client outside the appserver and that's not likely to work with what we have with EE 9 at the moment, because any jakarta.* namespace classes that get sent from the server to the client won't be understood by the client, which has javax.* namespace classes.

Reply:
I've added it to the exclude list, i'll push the changes. Not sure about your last comment; there are many tests which use EJB client from a standalone test case interacting with a server. Is this going to be a general problem?

Reply:
@rachmatowicz Sorry, I didn't get back to this for too long.

There are two aspects to my answer.

First, if we convert the codebase to jakarta namespace then this will no longer be a problem.

But, an important task for WF 26 is figuring out how to deal with remote EE 8 EJB clients accessing an EE > 8 server, as propagating java types to clients is part of remote EJB. This is something you, @chengfang, @tadamski, @ropalka and I need to talk about.